ui/docs/tutorial/download-task-artifacts.md

```diff
@@ -70,7 +70,7 @@ let runTask = async () => {
     
   console.log("Created task:\n" + JSON.stringify(result.status, null, 2));    
   console.log("Inspect it at:");    
-  console.log("https://tools.taskcluster.net/task-inspector/#" + taskId);    
+  console.log("https://tc.example.com/tasks/" + taskId);    
```

Comment:
This is fine though (most of these JS code snippets no longer work, to be honest)